geode-core/src/distributedTest/java/org/apache/geode/internal/cache/PartitionedRegionInvalidateDUnitTest.java

```diff
@@ -46,8 +47,9 @@ public PartitionedRegionInvalidateDUnitTest() {
     super();
   }
 
-  void createRegion(String name, boolean accessor, int redundantCopies) {
+  void createRegion(String name, boolean accessor, int redundantCopies, CacheWriter w) {
```

Comment:
This is for `CacheWriter`. I don't see a test for `CacheLoader`.

Reply:
This is actually a test meant to test a different use case. With our new changes, consecutive create region calls were failing due to our new update changing the loaderWriterByte, but this test was written to expect no writer to be set. There is no loader because this test was not previously using a loader, so we just fixed the bug with their writer.